great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -727,17 +728,40 @@ def build_map_metric_rule(
         )
     )
 
-    # Step-2: Declare "ParameterBuilder" for every metric of interest.
+    # Step-2: Declare "ParameterBuilder" for every relevant metric of interest.
 
-    column_values_unique_unexpected_count_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = (
-        DataAssistant.commonly_used_parameter_builders.get_column_values_unique_unexpected_count_metric_multi_batch_parameter_builder()
-    )
-    column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = (
-        DataAssistant.commonly_used_parameter_builders.get_column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder()
-    )
-    column_values_null_unexpected_count_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = (
-        DataAssistant.commonly_used_parameter_builders.get_column_values_null_unexpected_count_metric_multi_batch_parameter_builder()
-    )
+    parameter_builders: List[ParameterBuilder] = []
+
+    column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder_for_evaluations: Optional[
+        ParameterBuilder
+    ] = None
+
+    if map_metric_name == "column_values.unique":
+        column_values_unique_unexpected_count_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = (
+            DataAssistant.commonly_used_parameter_builders.get_column_values_unique_unexpected_count_metric_multi_batch_parameter_builder()
+        )
+        parameter_builders.append(
+            column_values_unique_unexpected_count_metric_multi_batch_parameter_builder_for_metrics
+        )
+    elif map_metric_name == "column_values.null":
+        column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = (
+            DataAssistant.commonly_used_parameter_builders.get_column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder()
+        )
+        column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder_for_evaluations = column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder_for_metrics
```

Comment:
@billdirks I think that this is a good idea -- I also have spotted many occurrences that trigger the `PyCharm` line length warning and would like to fix them.  Perhaps we can establish a standard for meaningful variable names that do not cause cognitive overload, while being as short as possible, and address all these situations at once?  We can file a JIRA item with this scope.  Thanks!

Reply:
I really think we should do this as we go along. I don't know why black is allowing one to commit at all since the default max line length is 88 chars. Maybe that is not turned on here for some reason?

Reply:
I see, black doesn't break up property access and we have really long names for our properties. You can manually do this by using parentheses though like:
```
(myobject.
   property1.
   property2
) = 5
```

Reply:
In this case we could do:
```
column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder_for_evaluations = (
    column_values_nonnull_unexpected_count_metric_multi_batch_parameter_builder_for_metrics
)
```